docs/CONTRIBUTING.md

```diff
@@ -9,7 +16,7 @@ license.
 
 # Responsible Disclosure of Security Vulnerabilities
 
-Please refer to our [Security Policy](./SECURITY.md) for our policy on
+Please refer to our [Security Policy](https://linkedin.github.io/venice/community_guides/security_vulnerability) for our policy on
```

Comment:
this gives me a 404. Similarly for some links you updated below. Is it same for you? I believe they will be available once our page is up, correct?

Reply:
Yes. It will be available once we merge the PR. The corresponding page on my fork is at: https://nisargthakkar.github.io/venice/community_guides/security_vulnerability